Classes/Mapper/AssetModelMetaDataMapper.php

```diff
@@ -83,7 +103,7 @@ public function mapMetaData(Asset $asset, MetaDataCollection $metaDataCollection
         $contextVariables = array_merge($this->defaultContextVariables, $metaDataCollection->toArray());
 
         if (isset($this->metaDataMappingConfiguration['title'])) {
-            $asset->setTitle((string)EelUtility::evaluateEelExpression($this->metaDataMappingConfiguration['title'], $this->eelEvaluator, $contextVariables));
+            $asset->setTitle(substr((string)EelUtility::evaluateEelExpression($this->metaDataMappingConfiguration['title'], $this->eelEvaluator, $contextVariables), 0, 255));
```

Comment:
Mhm, I think the cropping is better done using the eel expression, wehere it can easily be adjusted.